code/inc/gfx/gfx_glvibuffer.hpp

```diff
@@ -0,0 +1,145 @@
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**************************************************************************************************/
+
+#pragma once
+
+/*=================================================================================================
+** 1.  REFERENCES
+**===============================================================================================*/
+
+/*=================================================================================================
+** 2.  INCLUDE FILES
+**===============================================================================================*/
+
+#include "gfx/gfx_vibuffer.hpp"
+
+/*=================================================================================================
+** 3.  DECLARATIONS
+**
+** 3.1 Macros
+**===============================================================================================*/
+
+/*=================================================================================================
+** 3.2 Types and Classes
+**===============================================================================================*/
+namespace GFX
+{
+    namespace Render
+    {
+        /**************************************************************************************************/
+        /**
+        * \par Details: 
+        */
+        class GLVertexBuffer : public VertexIndexBuffer
+        {
+
+        public:
+            /*********************************/
+            // Public type definitions
+            /*********************************/
+
+            /*********************************/
+            // Public member variables
+            /*********************************/
+
+            /*********************************/
+            // Constructors/Destructor
+            /*********************************/
+
+            /**************************************************************************************************/
+            /**
+            * \brief The default constructor for the GLVertexBuffer.
+            */
+            GLVertexBuffer(void);
+
+            /*********************************/
+            // Public functions
+            /*********************************/
+
+            /**************************************************************************************************/
+            /**
+            * \brief 
+            * 
+            * \param[in] 
+            * 
+            * \retval 
+            */
```

Comment:
Fill in function comments

Reply:
Fixed https://github.com/tdiego1/worldweaver/pull/19/commits/1a0df967e70f1752177ebca99e4618e4ec06c26d